com.ibm.streamsx.inet/com.ibm.streamsx.inet.ftp/FTPCommand/FTPCommand_cpp.cgt

```diff
@@ -96,7 +134,7 @@ void MY_OPERATOR::prepareToShutdown() {
 
 // Tuple processing for non mutating ports
 void MY_OPERATOR::process(Tuple const & tuple, uint32_t port) {
-	SPLLOG(L_TRACE, "process(" << tuple << ", " << port << ")", debugAspect);
+	SPLAPPTRC(L_TRACE, "process(" << tuple << ", " << port << ")", debugAspect);
```

Comment:
Should this be protected with a test to see if we are logging at this level.

Also is this actually needed, does Streams trace tuples at fine levels anyway?